src/main/java/com/eternalcode/core/EternalCore.java

```diff
@@ -98,30 +101,40 @@
 
     private static final String VERSION = Bukkit.getServer().getClass().getName().split("\\.")[3];
 
-    @Getter private ConfigurationManager configurationManager;
-    @Getter private ScoreboardManager scoreboardManager;
-    @Getter private TeleportManager teleportManager;
-    @Getter private LiteCommands liteCommands;
-    @Getter private ChatManager chatManager;
-    @Getter private UserManager userManager;
-    @Getter private Scheduler scheduler;
+    @Getter
+    private ConfigurationManager configurationManager;
+    @Getter
+    private ScoreboardManager scoreboardManager;
+    @Getter
+    private TeleportManager teleportManager;
+    @Getter
+    private LiteCommands liteCommands;
+    @Getter
+    private ChatManager chatManager;
```

Comment:
Przywróc stary styl Getterów!

```java
    @Getter private ConfigurationManager configurationManager;
    @Getter private ScoreboardManager scoreboardManager;
    @Getter private TeleportManager teleportManager;
    @Getter private LiteCommands liteCommands;
    @Getter private ChatManager chatManager;
    @Getter private UserManager userManager;
    @Getter private Scheduler scheduler;

Reply:
Sory, mam plugin do intelki który to psuje